include/zenoh/api/config.hxx

```diff
@@ -37,11 +37,13 @@ class Config : public Owned<::z_owned_config_t> {
 
 #ifdef ZENOHCXX_ZENOHC
     /// @brief Create the configuration from the JSON file.
-    /// @param path path to the config file.
+    /// @param path path to the config file (see <a
+    /// href="https://zenoh.io/docs/manual/configuration/#configuration-files">config file documentation</a> for more
+    /// information).
     /// @param err if not null, the result code will be written to this location, otherwise ZException exception will be
     /// thrown in case of error.
     /// @return the ``Config`` object.
-    /// @note zenoh-c only.
+    /// @note Zenoh-c only.
```

Comment:
in changes above Zenoh was changed to zenoh, here is the opposite change. More places like this are below